lib/utils.py

```diff
@@ -20,6 +23,12 @@
 except:
     pass
 
+# This will work on x86-based machines
+try:
```

Comment:
There is incorrect comment. This code will works on any platform if you have mplayer installed, but we perfer omxplayer for arm and mplayer for x86, x86_64